extra-optenvw64/gcc+w64/spec

```diff
@@ -1,4 +1,3 @@
-VER=7.3.0
-REL=2
-SRCTBL="https://ftp.gnu.org/gnu/gcc/gcc-$VER/gcc-$VER.tar.xz"
-CHKSUM="sha256::832ca6ae04636adbb430e865a1451adf6979ab44ca1c8374f61fba65645ce15c"
+VER=9.3.1
+GITSRC="https://github.com/gcc-mirror/gcc"
```

Comment:
Please use the new ACBS [spec format](https://wiki.aosc.io/developer/packaging/acbs/spec-format/).

Reply:
> Please use the new ACBS [spec format](https://wiki.aosc.io/developer/packaging/acbs/spec-format/).

I cannot find anything outside the spec.

Do you want to switch to SRCS?